### ISSUE TITLE
RPi внезапное отключение экрана.

### ISSUE THREAD

**Issue by omletnet** [Ubuntu-Sway/Ubuntu-Sway-Remix#18]
Просто используешь UbuntuSR  и вдруг отключается экран(телевизор fhd). Почему? Не знаю.
Это случалось после подключения флешки, после нажатия клавиши клавиатуры и просто так, в бездействии.
Компьютер просто перестаёт выводит изображение.

**Comment by Sunderland93** [comment]
Да, проблему подтверждаю. Скажите, а при подключении флешки экран сразу отключается? или также спустя какое-то время простоя?

**Comment by omletnet** [comment]
> Да, проблему подтверждаю. Скажите, а при подключении флешки экран сразу отключается? или также спустя какое-то время простоя?

Из-за флешки было 2 раза (сразу, не думаю что после после простоя, т.к. и без неё экран тоже погасал).

**Comment by Sunderland93** [comment]
Пожалуйста проверьте с последним билдом https://downloads.ubuntusway.com/raspberrypi/latest/ubuntusway-22.04.2-stable-rpi-20230225.img.xz
